src/main/java/core/basesyntax/service/Reader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface Reader {
+    List<String> fileReader(String file);

```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names 
fix method names in all places